help/en/releasenotes/current-draft-note.shtml

```diff
@@ -445,8 +445,17 @@
         <a id="Dispatcher" name="Dispatcher"></a>
         <ul>
              <li>Provide more information when selecting traininfo file.</li>
+             <li>Add the ability to use an AdHoc transit, which is created dynamically between 2 blocks
```

Comment:
Is this one sentence that ends with "2 blocks"? If so, it seems that a dot is missing at the end of the row.